README.md

```diff
@@ -1,2 +1,24 @@
-# radiology
-Collaboration with Boston Children's Hospital, Boston University, and Red Hat to improve the scale and performance of the ChRIS project
+# Radiology
+The radiology project is a collaboration with Boston Children's Hospital, Boston University, and Red Hat to improve the scale and performance of [ChRIS](https://github.com/FNNDSC/).
+
+## ChRIS
+ChRIS (Children’s Research Integration System) is a web-based medical image platform that allows for various forms of medical image (Ex: MRIs) processing. ChRIS itself is comprised of multiple open source projects hosted on [GitHub](https://github.com/FNNDSC/) with the intention to make the research and capabilities available to other hospitals and institutions.
+
+## Massachusetts Open Cloud
+The [MOC](https://massopen.cloud/) is a cloud computing environment built in collaboration with 5 Massachusetts area universities (Boston University, Harvard, MIT, Northeastern, UMass) used primarily for educational and research purposes. The MOC, however, has the goal of expanding its footprint/usage by providing vertical solutions, such as the ChRIS project, to other institutions with similar needs.
+
+## Goals
+The desire of Boston Children’s Hospital, Boston University and Red Hat is to help improve the scale and efficiency of the ChRIS platform using the MOC and various Red Hat technologies including OpenShift and OpenStack. Today, processing a single set of images takes ~10 hrs. The goal is to get that down to a few mins to give the patients immediate feedback.
```

Comment:
Boston Children’s Hospital, Boston University, and Red Hat aim to improve the scale and efficiency of the ChRIS platform using the MOC and various Red Hat technologies, including OpenShift and OpenStack. Today, processing a single set of images takes about ten hours. The goal is to get that down to a few minutes to give patients quicker feedback.